usaspending_api/etl/tests/integration/test_load_table_to_delta.py

```diff
@@ -0,0 +1,87 @@
+"""Automated Integration Tests for the lifecycle of Delta Lake tables
+
+NOTE: Uses Pytest Fixtures from immediate parent conftest.py: usaspending_api/etl/tests/conftest.py
+"""
+from datetime import datetime
+
+from model_bakery import baker
+from pyspark.sql import SparkSession
+from pytest import mark
+from django.core.management import call_command
+
+from usaspending_api.etl.management.commands.create_delta_table import TABLE_SPEC
+
+
+def equal_datasets(ds1, ds2):
+    """Helper function to compare the two datasets. Note the column types of ds1 will be used to cast columns in ds2."""
+    datasets_match = True
+    for i, ds1_row in enumerate(ds1):
+        for k, val_1 in ds1_row.items():
+            val_1_type = type(val_1)
+            val_2 = ds2[i][k]
+            if not isinstance(val_1, type(None)) and not isinstance(val_1, type(val_2)):
+                val_2 = val_1_type(val_2)
+            if isinstance(val_1, datetime):
+                val_1 = val_1.replace(microsecond=0).replace(tzinfo=None)
+                val_2 = val_2.replace(microsecond=0).replace(tzinfo=None)
+            if val_1 != val_2:
+                raise Exception(f"Not equal: {val_1} {val_2} {val_1_type}")
+    return datasets_match
+
+
+def _verify_delta_table_loaded(spark: SparkSession, delta_table_name: str, s3_bucket: str):
+    """Generic function that uses the create_delta_table command to create the given table and assert it was created
+    as expected
+    """
+    # make the table and load it
+    call_command("create_delta_table", f"--destination-table={delta_table_name}", f"--spark-s3-bucket={s3_bucket}")
+    call_command("load_table_to_delta", f"--destination-table={delta_table_name}")
+
+    # get the postgres data to compare
+    model = TABLE_SPEC[delta_table_name]["model"]
```

Comment:
Nice use of adding `model` to the `TABLE_SPEC` here

Reply:
If this is to combat casting that is done during the ETL. Would it make sense to use the `custom_schema` property from the `TABLE_SPEC` to target specific casts.

Reply:
No the `model` is for database querying for testing.
`dummy_data = list(model.objects.order_by(partition_col).all().values())` 
and can be used to also retreive the meta table name if we wanted to remove the `source_table` attribute.